deploy-rh

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env bash
+
+##--------------------------------------------------------------------
+## Copyright (c) 2019 Dianomic Systems
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+##
+## Author: Mohd. Shariq
+##
+
+set -e
+
+# Variables
+__version__="1.5.2next"
+if [ -f "package.json" ]
+  then
+	__version__=`cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[" ,]//g'`
+fi
+
+FOGLAMP_GUI_VER=${__version__}
+
+# Colors
+CPFX="\033["
+
+CRESET="${CPFX}0m"          # Text Reset
+CERR="${CPFX}1;31m"
+CINFO="${CPFX}1;32m"
+CWARN="${CPFX}0;33m"
+
+DOC_ROOT="/usr/share/nginx/html"
+echo -e "${CWARN}This script is partially done, please track https://github.com/foglamp/foglamp-gui/issues/73 ${CRESET}"
+
+echo -e WARNING: "${CWARN} This script will remove all the contents of ${DOC_ROOT} ${CRESET}"
+read -p "Continue? Press n or N to exit." -n 1 -r
+echo    # (optional) move to a new line
+if [[ $REPLY =~ ^[Nn]$ ]]
+then
+    exit 0;
+fi
+
+machine_details() {
+  os=$(uname)
+  OS_NAME=`sudo cat /etc/*-release | grep PRETTY_NAME | cut -d= -f2-`
+  echo -e "${CINFO}Operating System: ${CRESET} ${OS_NAME} "
+  if [[ ${os} == "Darwin" ]]
+  then
+      echo -e Error:"${CERR} Script ${FOGLAMP_GUI_VER} is not compatible with macOS ${CRESET}"
```

Comment:
exit if NOT Red Hat|CentOS with `echo "This script is compatible with Red Hat Linux distributions only!"`